### PR TITLE
Create template for pull request

### PR DESCRIPTION
**Pull request by giovannism20**
### Pull Request Description
Since it's an open source project and he is growing is important to have standardization.

### Related Issue

Issue Number: #1974

### Pull Request Type

Please check the type of change your pull request introduces:

- Bugfix
- Feature
- Code style update (formatting, renaming)
- Refactoring (no functional changes, no api changes)
- Build related changes
- [X] Documentation content changes
- Other (please describe):

### Pull request checklist

- [X] Did you explain or provide a way of how can we test your code ?
- [X] If your pull request is related to frontend modifications provide a print of before and after screen
- [X] Did you provide a general summary of your changes ?
- [X] Try to limit your pull request to one type, submit multiple pull requests if needed

### CODE REVIEW

**Comment by vitormattos** [comment]
Suggestion to improve:
https://github.com/nextcloud/spreed/blob/main/.github/pull_request_template.md?plain=1

**Comment by giovannism20** [comment]
great suggestion @vitormattos i'll make these modifications!

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+### Pull Request Description
+
+
+### Related Issue
+
```
Comment (vitormattos) [inline .github/pull_request_template.md:5]:
```suggestion
<!--
If this PR is related to an issue, put here, if not, remove this block
-->
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- Build related changes
+- Documentation content changes
+- Other (please describe):
+
+### Pull request checklist
+
+- [ ] Did you explain or provide a way of how can we test your code ?
+- [ ] If your pull request is related to frontend modifications provide a print of before and after screen
+- [ ] Did you provide a general summary of your changes ?
+- [ ] Try to limit your pull request to one type, submit multiple pull requests if needed
```
Comment (vitormattos) [inline .github/pull_request_template.md:25]:
```suggestion
- [ ] Try to limit your pull request to one type, submit multiple pull requests if needed
- [ ] I implemented tests that cover my contribution
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Please check the type of change your pull request introduces:
+
+- Bugfix
+- Feature
+- Code style update (formatting, renaming)
+- Refactoring (no functional changes, no api changes)
+- Build related changes
+- Documentation content changes
+- Other (please describe):
```
Comment (vitormattos) [inline .github/pull_request_template.md:18]:
```suggestion
<!--
Please check the type of change your pull request introduces. Remove all that is unrelated and remove the comment block too, maintaining only the type of your PR:

- Bugfix
- Feature
- Code style update (formatting, renaming)
- Refactoring (no functional changes, no api changes)
- Build related changes
- Documentation content changes
- Other (please describe):
-->
```

**Review by vitormattos — COMMENTED** [review]
What you think about the suggestions?

**Review by vitormattos — APPROVED** [review]

**Pull request merged**